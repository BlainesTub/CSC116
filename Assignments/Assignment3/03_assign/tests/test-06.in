3
0 1 -14
0 2 -3
1 2 -9
