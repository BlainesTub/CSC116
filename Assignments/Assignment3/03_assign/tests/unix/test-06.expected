Input matrix. Size : 3 x 3
 -  : -14 :  -3 : 
-14 :  -  :  -9 : 
 -3 :  -9 :  -  : 

Input Adjacency list. Size : 3
From City: A -> (B,-14) (C,-3) 
From City: B -> (A,-14) (C,-9) 
From City: C -> (A,-3) (B,-9) 

Minimum Spanning Tree. Size : 3
From City: A -> (B,-14) 
From City: B -> (A,-14) (C,-9) 
From City: C -> (B,-9) 

Cost of original matrix: -26
Cost of minimum spanning tree: -23
